### ISSUE TITLE
How to I tell HTMX to ignore certain input fields when changing the URL?

### ISSUE THREAD

**Issue by cbmeeks** [bigskysoftware/htmx#1877]
I have a complex search bar that contains several input fields along with some dynamic input fields.

Some of these input fields have their own `hx-get` attribute in order to bring down "allowable" values from the server.  This works fine until I put one of these into a `form` containing `hx-replace-url`.

For example, look at the following code:

    <form autocomplete="off"
          hx-post="."
          hx-replace-url="true"
          hx-select="table">

        <div class="flex justify-center p-20 flex-col w-[30vw] mx-auto">
            <label for="search" class="font-semibold">Search</label>
            <input type="text" id="search" class="p-2 rounded-md relative" name="search" autocomplete="off"
                   hx-get="/items"
                   hx-swap="innerHTML"
                   hx-target="#items"
                   hx-trigger="keyup[target.value.length > 2]"
                   hx-select="div"
            >
            <div id="items" class="">Results</div>
        </div>

        <div class="" id="otherSearchFields">
            ... OTHER INPUT BOXES HERE SHOULD CHANGE URL...NOT THE "search" above
        </div>
    </form>

Notice the `search` input field has a name of `search` so that the server can add its value to the complex search login.  This search field should only be allowed to have the options brought down from the server.  I can make this work just fine except when it's inside the form.

If you look at the form, you will see it has `hx-replace-url`.  What's happening is that the URL is being changed to `/items?search=...`  but it needs to stay the same which is just `/` at the moment.

Basically I just want the fields in the div marked `otherSearchFields` to change the form's URL.

Is there an easy way to do this?

Thanks.

**Comment by Renerick** [comment]
It appears that you have encountered an undesired side effect of attribute inheritance. Try putting `hx-disinherit` attribute (https://htmx.org/attributes/hx-disinherit/) or `hx-replace-url="unset"` (https://htmx.org/docs/#inheritance) on the `/items` input

**Comment by cbmeeks** [comment]
I will give that a try.  Thanks!

**Comment by Renerick** [comment]
I'm going to close this issue as answered. Feel free to comment here or reopen it in case there are any issues
